doc/examples/viz_slice.py

```diff
@@ -194,12 +276,14 @@
 renderer.reset_camera()
 renderer.zoom(1.6)
 
-# window.show(renderer, size=(900, 600), reset_camera=False)
+
+# show_m.ren.add(panel_picking)
```

Comment:
When I close the previous window (Line 227 -`show_m.start()` ), it seems that `panel_picking` is deleted so I got an error when I reach here : 

`Traceback (most recent call last):
  File "viz_slice.py", line 280, in <module>
    show_m.ren.add(panel_picking)
  File "c:\users\skoudoro\devel\dipy\dipy\viz\window.py", line 84, in add
    actor.add_to_renderer(self)
  File "c:\users\skoudoro\devel\dipy\dipy\viz\ui.py", line 580, in add_to_renderer
    super(Panel2D, self).add_to_renderer(ren)
  File "c:\users\skoudoro\devel\dipy\dipy\viz\ui.py", line 86, in add_to_renderer
    iren = ren.GetRenderWindow().GetInteractor().GetInteractorStyle()
AttributeError: 'NoneType' object has no attribute 'GetInteractor' `

During this test, I just uncomented all `show_m.start()`

Reply:
Addressed in 6ef5736